target/cheri-common/cheri-compressed-cap/.gitrepo

```diff
@@ -5,8 +5,8 @@
 ;
 [subrepo]
 	remote = https://github.com/CTSRD-CHERI/cheri-compressed-cap
-	branch = master
-	commit = dfb460875b4c5650cd1e00493110ae75e1e4d2c7
-	parent = c1a4ca2d4527b63fe57ba66d9c9c48f72b5c8e1e
+	branch = morello
+	commit = 6e61e97621caa2132385a357a40b07b1c9ddcda0
+	parent = ca1edb1fcf2dde67d04419e49f5be1b3125ee358
```

Comment:
This is broken since you rebased, but I can deal with that later.

Reply:
I think the best thing to do is likely deal with cheri-compressed-cap separately, bump it normally in dev and then have a Morello PR to then use it.